x-pack/test/ingest_manager_api_integration/apis/fixtures/test_packages/all_assets/0.1.0/kibana/dashboard/sample_dashboard.json

```diff
@@ -11,6 +11,12 @@
     "title": "[Logs Sample] Overview ECS",
     "version": 1
   },
+  "references": [
+    { "id": "sample_visualization", "name": "panel_0", "type": "visualization" },
```

Comment:
Same as https://github.com/elastic/kibana/pull/82602#discussion_r518825901 and other `x-pack/test/ingest_manager_api_integration/apis/fixtures/test_packages/all_assets/*`. The sample dashboard JSON was incorrect as was missing references array and new migration failed when importing these sample dashboards